### PR TITLE
Add special standalone_with_prefix_path buildMode

### PR DESCRIPTION
**Pull request by jpbochi**
--buildMode is the replacement arg for --appMode

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ $ proton-bundler [action:optional] <--api> <--branch> <--flow> <--appMode> <--de
 
 ### flags
 - `--remote`: Build the current app from master via a git clone
-- `--branch`: **Mandatory** ex: deploy-settings 
+- `--branch`: **Mandatory** ex: deploy-settings
 - `--api`: Typeof branch to deploy (dev/beta/build/etc.)
 - `--flow`: Type of flow (_Usefull only for WebClient_)
 - `--no-lint`: Ignore lint task on deploy
-- `--appMode`: Type of bundle for the app (ex: standalone is an option for protonmail-settings)
+- `--buildMode`: Type of bundle npm task to run for the app (ex: "standalone" translates into `npm run bundle:standalone`)
```
Comment (dhoko) [inline Readme.md:50]: Can you keep the old one inside the doc but add **deprecated** next to it ?

File: lib/tasks/bundle.js [hunk 2]
```diff
@@ -63,11 +63,16 @@ function main({
             title: 'Build the application',
             async task(ctx = {}) {
                 const args = process.argv.slice(2);
-                if (appMode === 'standalone') {
+                if (buildMode === 'standalone') {
                     const output = await bash('npm', ['run', 'build:standalone', '--', ...args]);
                     ctx.outputBuild = output;
                     return true;
                 }
+                if (buildMode === 'standalone_with_prefix_path') {
```
Comment (dhoko) [inline lib/tasks/bundle.js]: Can you use this kind of syntax ? _standaloneWithPrefix_ etc. ?
We don't use this syntax inside projects.

Comment (jpbochi) [reply lib/tasks/bundle.js]: this sort of aligns with npm tasks, and there we have stuff like `check-types`. I don't care, though. Whichever you think is best.

Comment (dhoko) [reply lib/tasks/bundle.js]: check-types not check_types :grin:

**Pull request merged**